postman/README.md

```diff
@@ -242,6 +246,37 @@ Plugins will auto update alongside the CLI, but to trigger an update directly ru
 nim update
 ```
 
+## Manage Postman API Keys
+
+Key management command facilitates to have multiple postman API secrets to be stored as key value pair, and allows one of them to be used as current.
+It also allows to switch between the keys using `switch` command and thus becoming the current key.
+The current key is then picked up automatically without user needing to type it in manually.
+
+Adding the first key makes it current:
```

Comment:
```suggestion
Adding a key automatically makes it current.
```

Reply:
is it only the first key that becomes current or any added key?

Reply:
it's any added key.
thanks!